packages/console/src/ds-components/CardTitle/index.tsx

```diff
@@ -20,6 +20,12 @@ export type Props = {
   className?: string;
   /** If a beta tag should be shown next to the title. */
   isBeta?: boolean;
+  /**
+   * If a paywall tag should be shown next to the title. The value is the plan type.
+   *
+   * If not provided, no paywall tag will be shown.
+   */
+  paywall?: 'hobby' | 'pro';
```

Comment:
nit: I think we do have an enum for plans. Might consider referencing these values from the plan enum for the sake of SSOT?

Reply:
@xiaoyijun any sugguestions?

Reply:
We have a `ReservedPlanId` enum.

Reply:
let me take a look